--- a/net/minecraft/server/MinecraftServer.java
+++ b/net/minecraft/server/MinecraftServer.java
@@ -1065,6 +_,7 @@
         // Paper end - rewrite chunk system
 
         this.saveAllChunks(false, true, false, true); // Paper - rewrite chunk system
+        com.infernalsuite.asp.AdvancedSlimePaper.instance().awaitPendingSaves(); // Scissors - write the saves that world unloads queued
 
         this.isSaving = false;
         this.savedDataStorage.close();
